src/main/java/seedu/address/model/person/InternshipApplication.java

```diff
@@ -133,17 +203,46 @@ public String toString() {
                 .append("; Status: ")
                 .append(getStatus());
 
-        Set<Tag> tags = getTags();
-        if (!tags.isEmpty()) {
-            builder.append("; Tags: ");
-            tags.forEach(builder::append);
-        }
-
         if (!reviews.isEmpty()) {
             builder.append("; Review: ");
             reviews.forEach(builder::append);
         }
 
+        if (!programmingLanguages.isEmpty()) {
+            builder.append("; Programming Language: ");
+            programmingLanguages.forEach(builder::append);
+        }
+
+        if (!qualifications.isEmpty()) {
+            builder.append("; Qualification: ");
+            qualifications.forEach(builder::append);
+        }
+
+        if (location != null) {
+            builder.append("; Location: ")
+                    .append(getLocation());
+        }
+
+        if (salary != null) {
+            builder.append("; Salary: ")
+                    .append(getSalary());
+        }
+
+        if (!notes.isEmpty()) {
+            builder.append("; Note: ");
+            notes.forEach(builder::append);
+        }
+
+        if (rating != null) {
+            builder.append("; Rating: ")
+                    .append(getRating());
+        }
+
+        if (!reflections.isEmpty()) {
+            builder.append("; Reflection: ");
+            reflections.forEach(builder::append);
+        }
+
```

Comment:
I think adding more fields to toString method no longer meaningful, but still, good to have! Good job